Makefile

```diff
@@ -20,7 +20,7 @@ export ISTIO_GO
 SHELL := /bin/bash
 
 # Current version, updated after a release.
-VERSION ?= 0.5.0
+VERSION ?= 0.8-dev
```

Comment:
is this the right value here?

Reply:
The original value was not right either. The goal is to bring master to the same as release-0.8 for now.

/cc @costinm  who made this change.